api-reference/50 Common/utils/excelExporter/exportDataGrid(options).md

```diff
@@ -197,6 +197,15 @@ You can call this method at any point in your application. In the example below,
 
 ---     
 
+[note]
+
+The following restrictions apply when users export **DataGrid**:
+
+- [Detail rows](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/masterDetail/) are not exported
+- [Group rows](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Row/#rowType) are always exported expanded and the [isExpanded](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Row/#isExpanded) option is ignored.
```

Comment:
By "exported expanded" do you mean "exported in an expanded state"?

Reply:
Yes, exactly

Reply:
Thanks, please change the text